lighthouse-core/report/html/renderer/details-renderer.js

```diff
@@ -246,7 +246,7 @@ class DetailsRenderer {
     switch (heading.valueType) {
       case 'bytes': {
         const numValue = Number(value);
-        return this._renderBytes({value: numValue, granularity: 1});
```

Comment:
this is unrelated to multi, right? just needed for the eventual js audits?

Reply:
yeah I noticed it when I was making the js audits. I could pull this out as a one line change. It seems like a mistake to me.